### PR TITLE
feat: make flow run if all triggers pass

### LINKED ISSUE #918: add flow setting to only run if all triggers are successful
https://discord.com/channels/449576301997588490/867841013812035634/1134238693876572190

### PR DESCRIPTION
**Pull request by promise**
Fixes #918

### CODE REVIEW

**Review by djohts — APPROVED** [review]
lgtm

File: src/constants/editors/flows/steps/step4Details.ts
Comment (lacuna-exe) [inline src/constants/editors/flows/steps/step4Details.ts:1]: Meeting all triggers feels like an odd way to word this but I cannot think of a way to word it better.

Comment (promise) [reply src/constants/editors/flows/steps/step4Details.ts:1]: same. I found "logical conjunction" which is basically this but it's not user friendly enough idt.

**Review by aditya20-b — APPROVED** [review]
LGTM

**Review by wrac4242 — APPROVED** [review]

**Pull request merged**